tasks/DisableCpuFreqScaling.rb

```diff
@@ -0,0 +1,26 @@
+# For background on this read the following two links
+# https://wiki.gentoo.org/wiki/Power_management/Processor
+# https://wiki.archlinux.org/index.php/CPU_frequency_scaling
+
+class DisableCpuFreqScaling
+  include Genesis::Framework::Task
+
```

Comment:
I'm a bit paranoid and suggest adding `retries 1` so this does an immediate  hard fail instead of retrying.